File: 2016-08-15-WebViewExample/src/main/java/tech/thdev/webviewexample/webkit/listener/OnWebViewListener.java
package tech.thdev.webviewexample.webkit.listener;

/**
 * Created by tae-hwan on 8/14/16.
 */

public interface OnWebViewListener {

    void onScroll(int l, int t, int oldl, int oldt);

    void onFinish(String url);

    void onUrlChange(String url);

    void onProgressChanged(int newProgress);
}

File: app/src/main/java/tech/thdev/app/webkit/listener/OnWebViewListener.kt
package tech.thdev.app.webkit.listener

/**
 * Created by tae-hwan on 8/14/16.
 */
interface OnWebViewListener {
    fun onScroll(l: Int, t: Int, oldl: Int, oldt: Int)
    fun onFinish(url: String?)
    fun onUrlChange(url: String?)
    fun onProgressChanged(newProgress: Int)
}